tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java

```diff
@@ -0,0 +1,9 @@
+package com.quorum.tessera.jmeter;
```

Comment:
Is this meant to be part of this PR?

Reply:
But this PR isn't for the jmeter fix - it's only meant to be a refactoring of the code into a shared module isn't it?